.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "llvm"]
 	path = llvm
-	url = ../../circt/llvm.git
+	url = git@github.com:circt/llvm.git
```

Comment:
I know there was some discussion around this convention... @GeorgeLyon initiated the discussion in https://github.com/llvm/circt/pull/188 and we seemed to converge on the decision to use a relative path till we hit any issues.

@fabianschuiki is there some specific issue we hit here? I guess it assumes you are putting the copy of LLVM in a certain place that might not always hold. In my case, I don't even use the submodule, and prefer to keep a copy of LLVM in a very specific location I manage with CMake.

Reply:
Damn, thanks for pointing this out. This is an oversight on my side. I fixed this locally and just committed this without really thinking about it. Will revert this change; #188 makes total sense and changing that decision should at least go through review.

Reply:
Reverted in 0797984c. Thanks for bringing this to my attention! :+1:

Reply:
:+1: thanks!